src/test/scala-2.11/markets/participants/LiquiditySupplierSpec.scala

```diff
@@ -39,35 +39,38 @@ class LiquiditySupplierSpec extends TestKit(ActorSystem("LiquiditySupplierSpec")
     system.terminate()
   }
 
-  feature("A LiquiditySupplier should be able to schedule SubmitMarketOrder messages.") {
+  feature("A LiquiditySupplier should be able to schedule SubmitLimitOrder messages.") {
 
-    val market = TestProbe()
-    val prng = new Random(42)
-    val ticker = Agent(Tick(1, 1, Some(1), 1, 1))
     val tradable = TestTradable("GOOG")
+    val market = TestProbe()
+    val markets = mutable.Map[Tradable, ActorRef](tradable -> market.ref)
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [It's preferred to use immutable collections instead of mutable ones](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1705050673/issues/source?bid=2921361&fileBranchId=3046267#l46)